src/Common/src/System/Net/Logging/NetEventSource.Common.cs

```diff
@@ -48,7 +48,16 @@ namespace System.Net
 #if NET46    
     [SecuritySafeCritical]
 #endif
-    internal sealed partial class NetEventSource : EventSource
+
+    // Unblock reflection in ILC mode by declaring types and methods as public.
+    // An alternative would be to unblock reflection via rd.xml/csproj changes, but that would
+    // result in a size hit for all of the metadata.
+#if uap
+    public
```

Comment:
An rd.xml change isn't needed if we go this route (I've validated this by collecting logs during UAPAOT test runs). I haven't tried an rd.xml-based approach, but that would incur in larger metadata sizes.